package.json

```diff
@@ -47,6 +47,7 @@
     "postcss": "^6.0.22",
     "postcss-cli": "^5.0.0",
     "prettier-eslint-cli": "^4.7.1",
+    "eslint-plugin-react": "7.7.0",
```

Comment:
please remove this package. `@masterclass/eslint-config` already includes this package